tutorials/quickstart/Solution/ClientUI/Properties/launchSettings.json

```diff
@@ -0,0 +1,30 @@
+﻿{
+  "$schema": "http://json.schemastore.org/launchsettings.json",
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:5359",
+      "sslPort": 0
+    }
+  },
+  "profiles": {
+    "IIS Express": {
+      "commandName": "IISExpress",
+      "launchBrowser": true,
+      "launchUrl": "home",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
+    },
```

Comment:
Was this just boilerplate? I wonder if we should remove the IIS Express option so that there's really only one thing they can do.

Reply:
It is boilerplate.  I tested with both Kestrel and IISExpress and they both work fine.  In my experience, some of our less terminal-savvy counterparts will prefer IISExpress and Visual Studio.  We can remove it if you want.